### PR TITLE
[BUGFIX] Make flash example work with ES6

### PR DESCRIPTION
**Pull request by linawolf**
I will take care of Ajax in a follow up

references https://github.com/TYPO3-Documentation/t3docs-examples/issues/142

### CODE REVIEW

File: Resources/Private/Templates/Module/Flash.html [hunk 1]
```diff
@@ -12,24 +18,43 @@ <h1>
...
          {f:translate(key:'function_flash_content_elements')}
       </f:link.action>
    </p>
+   <p>
+      <f:translate key="function_flash_javascript"/>
+   </p>
+   <p>
+      <a href="#"
+         id="displayJavaScriptFlashMessage"
+         title=" {f:translate(key:'function_flash_javascript_button')}"
```
Comment (simonschaufi) [inline Resources/Private/Templates/Module/Flash.html:48]: One space too much

File: Resources/Private/Templates/Module/Flash.html [hunk 2]
```diff
@@ -12,24 +18,43 @@ <h1>
...
+         class="btn btn-default action-button">
+         {f:translate(key:'function_flash_javascript_button')}
+      </a>
+      <a href="#"
+         id="displayJavaScriptFlashAjaxMessage"
+         title=" {f:translate(key:'function_flash_javascript_button')}"
```
Comment (simonschaufi) [inline Resources/Private/Templates/Module/Flash.html:54]: Same

File: Resources/Private/Templates/Module/Flash.html (hunk 1 above)
Comment (linawolf) [inline Resources/Private/Templates/Module/Flash.html:48]:
```suggestion
         title="{f:translate(key:'function_flash_javascript_button')}"
```

File: Resources/Private/Templates/Module/Flash.html (hunk 2 above)
Comment (linawolf) [inline Resources/Private/Templates/Module/Flash.html:54]:
```suggestion
         title="{f:translate(key:'function_flash_javascript_button')}"
```

File: Resources/Public/JavaScript/flash-ajax-demo.js [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
- * JS code for extension "examples" BE module
+ * Module: @t3docs/examples/flash-message-demo
+ * JavaScript code for extension "examples" backend module
  */
+
+import Notification from "@typo3/backend/notification.js";
+
+class FlashAjaxDemo {
+	constructor() {
+		// todo: Resore Ajax example, see commented out Code below
```
Comment (brotkrueml) [inline Resources/Public/JavaScript/flash-ajax-demo.js:10]:
```suggestion
		// todo: Restore Ajax example, see code commented out below
```

File: Resources/Private/Templates/Module/Flash.html [hunk 4]
```diff
@@ -12,24 +18,43 @@ <h1>
...
+      <f:translate key="function_flash_javascript"/>
+   </p>
+   <p>
+      <a href="#"
+         id="displayJavaScriptFlashMessage"
+         title="{f:translate(key:'function_flash_javascript_button')}"
+         class="btn btn-default action-button">
+         {f:translate(key:'function_flash_javascript_button')}
+      </a>
+      <a href="#"
```
Comment (brotkrueml) [inline Resources/Private/Templates/Module/Flash.html:52]:
```suggestion
            <!-- Handled by Resources/Public/JavaScript/flash-ajax-demo.js -->
            <a href="#"
```

File: Resources/Private/Templates/Module/Flash.html [hunk 5]
```diff
@@ -12,24 +18,44 @@ <h1>
...
          title=" {f:translate(key:'function_flash_pages')}"
          class="btn btn-default action-button">
          {f:translate(key:'function_flash_content_elements')}
       </f:link.action>
    </p>
+   <p>
+      <f:translate key="function_flash_javascript"/>
+   </p>
+   <p>
+      <a href="#"
```
Comment (brotkrueml) [inline Resources/Private/Templates/Module/Flash.html:46]:
```suggestion
      <!-- Handled by Resources/Public/JavaScript/flash-message-demo.js -->
      <a href="#"
```

**Review by brotkrueml — APPROVED** [review]

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**